livingpark_utils/scripts/mri_metadata.py

```diff
@@ -41,7 +41,7 @@
 import pandas as pd
```

Comment:
where did `pd_status` go?

Reply:
The notebook was modified, however the script is only generated when merging the PR. The generation is done through a [GitHub action](https://github.com/LivingPark-MRI/livingpark-utils/blob/main/.github/workflows/gen_scripts.yaml).